docs/api/cases/cases-api-get-reporters.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[cases-api-get-reporters]]
+== Get reporters API
+++++
+<titleabbrev>Get reporters</titleabbrev>
+++++
+
+Returns all case reporters (users who opened cases).
```

Comment:
Since "reporters" is already in the title, I'd be inclined to lean into the second half of this description. For example:
```
Returns information about the users who opened cases.
```